File: app/src/main/java/co/tpcreative/supersafe/model/Authorization.java
package co.tpcreative.supersafe.model;
import java.io.Serializable;

public class Authorization implements Serializable {
    public String device_id;
    public String device_type;
    public String manufacturer;
    public String name_model ;
    public String version;
    public String versionRelease;
    public String session_token;


     public String user_id;
     public boolean active;
     public String created_date;
     public String updated_date;
     public String date_time;
     public String refresh_token;
     public String public_key;
}

File: app/src/main/java/co/tpcreative/supersafe/model/Authorization.kt
package co.tpcreative.supersafe.model
import java.io.Serializable

open class Authorization : Serializable {
    var device_id: String? = null
    var device_type: String? = null
    var manufacturer: String? = null
    var name_model: String? = null
    var version: String? = null
    var versionRelease: String? = null
    var session_token: String? = null
    var user_id: String? = null
    var active = false
    var created_date: String? = null
    var updated_date: String? = null
    var date_time: String? = null
    var refresh_token: String? = null
    var public_key: String? = null
}